test/specs/requests.spec.js

```diff
@@ -157,6 +157,26 @@ describe('requests', function () {
     });
   });
 
+  it('When the file protocol is initiated, it should reolve when response.status is equal to 0', function (done) {
```

Comment:
I suggest to change the case description as, 'should resolve when the response status is 0 (i.e. requesting with file protocol)'.